pacai/util/probability.py

```diff
@@ -50,8 +46,8 @@ def nSample(distribution, values, n):
     return samples
 
 def sample(distribution, values = None):
-    if type(distribution) == counter.Counter:
-        items = sorted(list(distribution.items()))
+    if type(distribution) == dict:
+        items = sorted(distribution.items())
         distribution = [i[1] for i in items]
         values = [i[0] for i in items]
 
```

Comment:
Check that `values` is none here.
After that, also check if `distribution` and `values` is the same size (two different checks).